homeassistant/components/sensor/sytadin.py

```diff
@@ -0,0 +1,130 @@
+"""
+Support for Sytadin Traffic, French Traffic Supervision.
+
+Systadin website : http://www.sytadin.fr
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.sytadin/
+"""
+import logging
+
+from datetime import timedelta
+import voluptuous as vol
+
+from homeassistant.const import (LENGTH_KILOMETERS, CONF_NAME)
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['lxml==4.0.0']
+
+SYSTADIN = 'http://www.sytadin.fr/sys/barometres_de_la_circulation.jsp.html'
+
+TRAFFIC_JAM_XPATH = '//*[@id="main_content"]/div[1]/div/span[3]/text()'
+MEAN_VELOCITY_XPATH = '//*[@id="main_content"]/div[2]/div/span[3]/text()'
+CONGESTION_XPATH = '//*[@id="main_content"]/div[3]/div/span[3]/text()'
+
+TRAFFIC_JAM_REGEX = '([0-9]+)'
+MEAN_VELOCITY_REGEX = '([0-9]+)'
+CONGESTION_REGEX = '([0-9]+.[0-9]+)'
```

Comment:
It seems that you only need one regex. Something like `(\d*\.\d+|\d+)` should work.

Reply:
fixed